src/terminal/terminal.rs

```diff
@@ -149,15 +151,18 @@ where
             viewport_area,
             last_known_size: size,
             last_known_cursor_pos: cursor_pos,
+            frame_count: 0,
         })
     }
 
     /// Get a Frame object which provides a consistent view into the terminal state for rendering.
     pub fn get_frame(&mut self) -> Frame {
+        let count = self.frame_count.wrapping_add(1);
```

Comment:
It looks like you're incrementing twice. Here and in flush. Am I mistaken?

Reply:
Yes, but this adds and stores the result in the `Frame` struct which is passed to the `FnOnce` callback the user provides. The second increments the field in the `Terminal` struct.

I can combine the two by doing `self.frame_count = self.frame_count.wrapping_add(1)` just in this function.

Reply:
Simplified implementation in https://github.com/ratatui-org/ratatui/pull/766/commits/842ef80fb7acca1f8660b886864d70ac649507f5.